.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pipeline
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+env:
+  IMAGE_TAG: ${{ github.sha }}
```

Comment:
Can't see where this is used, unless it is used in 'retag-and-push.sh'. Might be worth cleaning up if it isn't mind.